inbox/mailsync/backends/gmail.py

```diff
@@ -55,174 +55,6 @@
 MAX_DOWNLOAD_COUNT = 1
 
 
-class GmailSyncMonitor(ImapSyncMonitor):
```

Comment:
I just switched the order of classes here so I can access `GmailFolderSyncEngine` from `GmailSyncMonitor`